adr/recipes/activedata_usage.py

```diff
@@ -8,10 +8,11 @@
 
 from __future__ import print_function, absolute_import
 
-from ..recipe import execute_query
+from ..query import run_query
 
 
-def run(args):
+def run(args, config):
```

Comment:
What if the signature of this function is:

    def run(config, **context):

My reasoning is that most recipes seem to call `vars(args)` anyway, so this way the context is already a `dict`. Also this matches the `run_query` signature (so it should be easier to follow around).

What do you think?

Reply:
Yes it will be simpler, but I need more time to work on `extract_arguments()` to see how it work with args as a dict (and another problem when we set attribute value for query_args instead of args).